tt_metal/impl/device/device.cpp

```diff
@@ -280,14 +310,14 @@ void Device::compile_command_queue_programs() {
     const uint32_t num_tensix_command_slots = 2;
     const uint32_t num_eth_command_slots = 1;
 
-    uint32_t cmd_start_tensix = get_command_start_l1_address(false);
-    uint32_t data_section_addr_tensix = get_data_section_l1_address(false);
-    uint32_t producer_data_buffer_size_tensix = get_producer_data_buffer_size(false);
-    uint32_t consumer_data_buffer_size_tensix = get_consumer_data_buffer_size(false);
+    uint32_t cmd_start_tensix = get_command_start_l1_address(false, false);
```

Comment:
these apis and `compile_command_queue_programs` changed significantly since tunnelling to remote device was pushed. I can work with you if you run into issues with the rebase

Reply:
rebased.